config/development.json

```diff
@@ -78,6 +78,7 @@
 		"jetpack/connect/remote-install": true,
 		"jetpack/connect/mobile-app-flow": true,
 		"jetpack/connect/woocommerce": true,
+		"jetpack/connect/site-questions": true,
```

Comment:
We'll need to add this flag to all other env configs as well to ensure that the steps stay visible everywhere when this PR is merged.